someip2val/proto/kuksa_data_broker/kuksa/val/v1/types.proto

```diff
@@ -0,0 +1,286 @@
+/********************************************************************************
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
```

Comment:
We do not have a NOTICE file in kuksa.val.feeders, right?

Reply:
We have the same NOTICE reference in every header, better fix it in separate PR